client/webserver/site/src/js/forms.ts

```diff
@@ -326,7 +326,7 @@ export class NewWalletForm {
         break
       }
     }
-    const noWalletPWNeeded = walletDef.noauth || walletDef.seeded || Boolean(this.current.asset.token)
+    const noWalletPWNeeded = walletDef.seeded || Boolean(this.current.asset.token)
```

Comment:
I think the reason this was done wrong is because the variable `noWalletPWNeeded` was named incorrectly at conception.  It should really be `isCreated` afaict.

The other thing is that `page.auth` is badly named too.  It is a `div` that includes both the `newWalletPassBox` **and** the `submitAdd` button.

![image](https://user-images.githubusercontent.com/9373513/235233888-857ad357-f8ad-496e-9953-81bd3fea914d.png)

![image](https://user-images.githubusercontent.com/9373513/235233904-deaecfa4-6d0b-4d1b-8cb7-422782ef85fd.png)

Almost every time I come to this code, I have to go figure out what `page.auth` is and why that should be shown when `noWalletPWNeeded` (because it actually still includes the Add/Create!) button.

So `page.auth` -> `page.walletPassAndSubmitBttn`?

Both suggested changes are code churn, but this code is hard to reason about presently.

Reply:
I renamed `noWalletPWNeeded` to `displayCreateBtn` cuz that's what it does basically. IMO `isCreated` raises more questions(what is created?).